app/src/main/java/org/schabi/newpipe/fragments/list/BaseListFragment.java

```diff
@@ -350,12 +350,18 @@ protected void showStreamDialog(final StreamInfoItem item) {
         if (context == null || context.getResources() == null || activity == null) {
             return;
         }
-
         final List<StreamDialogEntry> entries = new ArrayList<>();
 
         if (PlayerHolder.getInstance().getType() != null) {
-            entries.add(StreamDialogEntry.enqueue);
+            entries.addAll(Arrays.asList(
+                    StreamDialogEntry.enqueue
+            ));
         }
+
+        if (PlayerHolder.getInstance().getQueueSize() > 1) {
```

Comment:
Move this if above inside the other if just above, as if there is no player there surely isn't there a queue